.github/workflows/manual-deployment.yml

```diff
@@ -3,6 +3,13 @@ name: Manual deployment
 on:
   workflow_dispatch:
     inputs:
+      service-name:
+        required: true
+        type: choice
+        options:
+          - comms-gatekeeper
+          - comms-gatekeeper-local-multiplayer
```

Comment:
local-multiplayer is something deployable to the cloud? 🤔

Reply:
we need it to use in the creators hub